pages/learn/welcome/introduction.md

```diff
@@ -26,21 +27,4 @@ Check here for details on all the tools we provide for managing your device flee
 
 Haven't found what you're looking for? Stop by our [FAQ][faq] and [Reference][reference] sections, or better yet, use the search in the upper-left corner. And if you still have questions, [help][help] is always available!
 
-You can also check out our [public roadmap][roadmap] and share your ideas and feature requests!
-
-[balena]:{{ $links.mainSiteUrl }}/
-[git]:https://git-scm.com/
-[docker]:https://www.docker.com/
-[rasppi]:https://www.raspberrypi.org/
-[nuc]:https://www.intel.com/content/www/us/en/products/boards-kits/nuc.html
-[primer]:/learn/welcome/primer
-[concepts]:/learn/welcome/concepts
-[getting-started]:/learn/getting-started
-[develop]:/learn/develop
-[deploy]:/learn/deploy
-[manage]:/learn/manage
-[faq]:/faq
-[reference]:/reference
-[help]:/learn/welcome/support
-[roadmap]:https://trello.com/b/9CSzdCXQ/balena-product-roadmap
```

Comment:
Hi @Conanza00, If you go to the introduction page right now (https://www.balena.io/docs/learn/welcome/introduction/), you'll see these bullet points aren't rendered in the actual docs, so no worries about removing them. They're there for our docs engine to pick up. When you specify `[link-variable-name]:/url/for/link` in this list, the docs engine will pick it up and look for anything in the same README that's labeled `[link-displayed-name][link-variable-name]`, and link them accordingly.